datayoga/blocks/rename_field/block.py

```diff
@@ -11,8 +11,8 @@ class Block(DyBlock):
     def init(self):
         logger.debug(f"Initializing {self.get_block_name()}")
 
-    def run(self, data: Any, context: Context = None) -> Any:
+    def run(self, data: List[Dict[str, Any]], context: Context = None) -> Any:
```

Comment:
return type should also be List[Dict[str, Any]]

Reply:
done